### PR TITLE
Remove Flavor Text for Identity Banned Players

### PR DESCRIPTION
**Pull request by Angelite191**
Literally what it says on the tin. Please review this carefully as I don't know how to test this locally.

### CODE REVIEW

File: code/modules/mob/new_player/new_player.dm [hunk 1]
```diff
@@ -406,6 +406,7 @@ Round Duration: [round(hours)]h [round(mins)]m<br>"}
 		new_character.gender = pick(MALE, FEMALE)
 		client.prefs.real_name = random_name(new_character.gender)
 		client.prefs.randomize_appearance_for(new_character)
+		client.prefs.flavor_text = ()
```
Comment (ComicIronic) [inline code/modules/mob/new_player/new_player.dm]: = ""

Comment (d3athrow) [inline code/modules/mob/new_player/new_player.dm]: ONE JOB

File: code/modules/mob/living/carbon/human/emote.dm [hunk 2]
```diff
@@ -728,4 +728,9 @@
 	set desc = "Sets an extended description of your character's features."
 	set category = "IC"
 
-	flavor_text =  copytext(sanitize(input(usr, "Please enter your new flavour text.", "Flavour text", null)  as text), 1)
+	if(appearance_isbanned(user))
```
Comment (ComicIronic) [inline code/modules/mob/living/carbon/human/emote.dm]: usr

**Comment by Angelite191** [comment]
!!SUCCESS!!

File: html/changelogs/angeliteentyshak_idban.yml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+author: Angelite Entyshak
+
+delete-after: True
+
+changes:
+	- bugfix: Identity banned player can no longer have flavortext.
```
Comment (d3athrow) [inline html/changelogs/angeliteentyshak_idban.yml]: 2 spaces not a tab between the new line and the hyphen.

**Pull request merged**